modules/grandpa/src/lib.rs

```diff
@@ -270,7 +271,7 @@ pub mod pallet {
 	/// Hash of the best finalized header.
 	#[pallet::storage]
 	pub type BestFinalized<T: Config<I>, I: 'static = ()> =
-		StorageValue<_, BridgedBlockHash<T, I>, ValueQuery>;
+		StorageValue<_, (BridgedBlockNumber<T, I>, BridgedBlockHash<T, I>), OptionQuery>;
```

Comment:
The storage value type changed from `BridgedBlockHash<T, I>` to a tuple `(BridgedBlockNumber<T, I>, BridgedBlockHash<T, I>)`. Need dig deeper to see if it breaks the storage on the chain.